Android-Sample/app/README.md

```diff
@@ -0,0 +1,94 @@
+# NetworkQualityTest
+
+The `NetworkQualityTest` class is a utility for testing network quality and gathering statistics
+during video streaming sessions using the OpenTok platform. It allows you to monitor various network
+and media statistics, such as video bitrate, audio quality, round-trip time, and more.
```

Comment:
Add a link to the docs on the callback methods (see my comments on the main README).